appliance/mongodb/process.go

```diff
@@ -770,7 +770,7 @@ func (p *Process) userExists() (bool, error) {
 	if err := session.Run(bson.M{"usersInfo": bson.M{"user": "flynn", "db": "admin"}}, &entry); err != nil {
 		return false, err
 	}
-	return entry.Retval != nil, nil
+	return entry.Retval == nil, nil
```

Comment:
Actually you might be right.. I am going to take a closer look at the query because something isn't quite right.